alerts/flink/README.md

```diff
@@ -0,0 +1,27 @@
+# Flink Alerts for Ops Agent
+
+### Notification Channels
+For all alerts, a notification channel needs to be set up or the alert will fire silently.
+
+### User Labels
+User labels can be used for these policies by modifying the userLabels fields of the policies. i.e.
+
+```json
+{ 
+  "userLabels": {
+    "datacenter": "central"
+  }
+}
+```
+
+## High job restarts alert
+When the job restarts delta value exceeds a threshold value, that indicates the flink jobs are not running in a healthy state.
```

Comment:
Thanks for the descriptions, these are great. Could you please also include in each description what threshold value is used in the sample alert policy

Reply:
added in default in description readme and json description.